package.json

```
@@ -79,6 +79,7 @@
     "@emotion/eslint-plugin": "11.11.0",
     "@faker-js/faker": "9.6.0",
     "@formatjs/cli": "6.6.1",
+    "@playwright/test": "^1.52.0",
```

let's pin that bad boy
```suggestion
    "@playwright/test": "1.52.0",
```